k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -1754,17 +1754,17 @@ public void limitToDisplayableFolders(LocalSearch search) {
         switch (displayMode) {
             case FIRST_CLASS: {
                 // Count messages in the INBOX and non-special first class folders
-                search.and(Searchfield.DISPLAY_CLASS, FolderClass.FIRST_CLASS.name(),
+                search.and(SearchSpecification.SearchField.DISPLAY_CLASS, FolderClass.FIRST_CLASS.name(),
```

Comment:
Please add an import statement for `SearchField` to avoid unnecessarily long expressions.
This is also an issue in several other files.